### ISSUE TITLE
pip installation error on Mac

### ISSUE THREAD

**Issue by ogencoglu** [neuro-ml/imops#45]
Here is the full log of `pip install imops` on Mac, Python3.11, virtualenv:

```
Building wheels for collected packages: imops
  Building wheel for imops (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for imops (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [55 lines of output]
      /private/var/folders/rp/d8kwybr118xd68k4qcxw93vw0000gn/T/pip-build-env-f33rnjg3/overlay/lib/python3.11/site-packages/setuptools/config/_apply_pyprojecttoml.py:73: _WouldIgnoreField: `dependencies` defined outside of `pyproject.toml` would be ignored.
      !!
      
              ********************************************************************************
              ##########################################################################
              # configuration would be ignored/result in error due to `pyproject.toml` #
              ##########################################################################
      
              The following seems to be defined outside of `pyproject.toml`:
      
              `dependencies = ['numpy<2.0.0,>=1.7; python_version < "3.8"', 'numpy<2.0.0,>=1.22; python_version >= "3.8"', 'scipy<2.0.0,>=1.0', 'dataclasses; python_version < "3.7"', 'scikit-image', 'connected-components-3d', 'fastremap']`
      
              According to the spec (see the link below), however, setuptools CANNOT
              consider this value unless `dependencies` is listed as `dynamic`.
      
              https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
      
              For the time being, `setuptools` will still consider the given value (as a
              **transitional** measure), but please note that future releases of setuptools will
              follow strictly the standard.
      
              To prevent this warning, you can list `dependencies` under `dynamic` or alternatively
              remove the `[project]` table from your file and rely entirely on other means of
              configuration.
      
              By 2023-Oct-30, you need to update your project and remove deprecated calls
              or your builds will no longer be supported.
              ********************************************************************************
      
      !!
        _handle_missing_dynamic(dist, project_table)
      running bdist_wheel
      running build
      running build_py
      running build_ext
      cythoning imops/src/_backprojection.pyx to imops/src/_backprojection.c
      cythoning imops/src/_fast_backprojection.pyx to imops/src/_fast_backprojection.c
      cythoning imops/src/_measure.pyx to imops/src/_measure.c
      cythoning imops/src/_fast_measure.pyx to imops/src/_fast_measure.c
      cythoning imops/src/_morphology.pyx to imops/src/_morphology.c
      cythoning imops/src/_fast_morphology.pyx to imops/src/_fast_morphology.c
      cythoning imops/src/_numeric.pyx to imops/src/_numeric.c
      cythoning imops/src/_fast_numeric.pyx to imops/src/_fast_numeric.c
      cythoning imops/src/_radon.pyx to imops/src/_radon.c
      cythoning imops/src/_fast_radon.pyx to imops/src/_fast_radon.c
      cythoning imops/src/_zoom.pyx to imops/src/_zoom.c
      cythoning imops/src/_fast_zoom.pyx to imops/src/_fast_zoom.c
      building 'imops.src._backprojection' extension
      creating build
      creating build/temp.macosx-13-arm64-cpython-311
      creating build/temp.macosx-13-arm64-cpython-311/imops
      creating build/temp.macosx-13-arm64-cpython-311/imops/src
      clang -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk -DNPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION -I/private/var/folders/rp/d8kwybr118xd68k4qcxw93vw0000gn/T/pip-build-env-f33rnjg3/overlay/lib/python3.11/site-packages/numpy/core/include -I/Users/ouz/Desktop/external_data/floorplan_curation/fp_curation_venv/include -I/opt/homebrew/opt/python@3.11/Frameworks/Python.framework/Versions/3.11/include/python3.11 -c imops/src/_backprojection.c -o build/temp.macosx-13-arm64-cpython-311/imops/src/_backprojection.o -fopenmp
      clang: error: unsupported option '-fopenmp'
      error: command '/usr/bin/clang' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for imops
Failed to build imops
ERROR: Could not build wheels for imops, which is required to install pyproject.toml-based projects
```

Any help would be appreciated.

**Comment by vovaf709** [comment]
For some reason we didn't provide wheels for your configuration yet. I'll fix this asap, it won't take more than a couple of days I suppose.

The problem is that in your case `clang` is used for compilation of produced C code instead of `gcc`. Do you have `gcc` installed?

**Comment by ogencoglu** [comment]
Thanks!
No, I do not have gcc but I can install it with Homebrew if needed.

**Comment by vovaf709** [comment]
Fixed by #63
